jormungandr/src/jrpc/mod.rs

```diff
@@ -22,7 +24,12 @@ pub async fn start_jrpc_server(config: Config, _context: ContextLock) {
 
     #[cfg(feature = "evm")]
     modules
-        .merge(eth_block_info::eth_get_blocks_info_module(_context))
+        .merge(eth_block_info::eth_get_blocks_info_module(_context.clone()))
+        .unwrap();
+
+    #[cfg(feature = "evm")]
+    modules
```

Comment:
nit: I think you can have a code block and use only one conditional compilation directive